samples/read-post/README.md

```diff
@@ -0,0 +1,7 @@
+# Read POST body
+
+This sample illustrates how to read the body of an HTTP POST request.
+
+To test different kinds of data, use `curl` with `-X POST` and the `-H` flag to set the `content-type` header.
```

Comment:
It would be nice to have some curl commands to paste into the terminal 

```
curl http://localhost:3000 --header "Content-Type: application/json" --data '{"greeting": "hello world"}'
```

```
curl http://localhost:3000 --header "Content-Type: image/cat" --data @hobbes.jpg
```